maps/mjolnir/bos/structures/furniture.dm

```diff
@@ -0,0 +1,97 @@
+///
+//CLOSETS
+///
+
+/obj/structure/closet/mjolnir
+	icon = 'maps/mjolnir/bos/icons/obj/structures/closets/closet.dmi'
+	closet_appearance = null
+
+/obj/structure/closet/mjolnir/wooden
+	name = "wooden closet"
+	desc = "It's a basic wooden closet."
+	icon = 'maps/mjolnir/bos/icons/obj/structures/closets/small.dmi'
+
+/obj/structure/closet/mjolnir/wooden/big
+	name = "big wooden closet"
+	icon = 'maps/mjolnir/bos/icons/obj/structures/closets/huge.dmi'
+
+
+///
+//BEDS
+///
+/obj/structure/bed/mjolnir
+	name = "bunk"
+	desc = "This is used to lie in, sleep in or strap on."
+	icon = 'maps/mjolnir/bos/icons/obj/structures/furniture/beds.dmi'
+	icon_state = "bunk"
+	base_icon = "bunk"
+
+/obj/structure/bed/mjolnir/double
+	name = "double bunk"
+	icon_state = "bunk_double"
+	base_icon = "bunk_double"
+
+/obj/structure/bed/mjolnir/wooden
+	name = "wooden bunk"
+	icon_state = "bunk_wooden"
+	base_icon = "bunk_wooden"
+
+///
+//DRAWERS
+///
+/obj/structure/filingcabinet/mjolnir
+	name = "steel cabinet"
+	desc = "A large cabinet with drawers."
+	icon = 'maps/mjolnir/bos/icons/obj/structures/furniture/drawers.dmi'
+	icon_state = "steel"
+
+///
+//TABLES
+///
+
+//FIXED//
+
+/obj/structure/table/rack/mjolnir
+	name = "wooden cabinet"
+	desc = "A large cabinet with drawers."
+	icon = 'maps/mjolnir/bos/icons/obj/structures/furniture/tables.dmi'
+	icon_state = "fixed"
+
+/obj/structure/table/rack/mjolnir/table
+	name = "wooden table"
+	desc = "A wooden table"
+	icon_state = "fixed4"
+
+/obj/structure/table/rack/mjolnir/roundtable
```

Comment:
`mjolnir/roundtable - > mjolnir/table/round`? For the sake of the better (shit)code